iml-timer.service

```
@@ -0,0 +1,17 @@
+[Unit]
+Description=IML Timer Service
+PartOf=iml-manager.target
+After=rabbitmq-server.service
+After=iml-settings-populator.service
+Requires=iml-settings-populator.service
+
+[Service]
+Type=simple
+Environment=RUST_LOG=info,sqlx::query=warn
```

```suggestion
Environment=RUST_LOG=info
```

`sqlx` is not being used here